index.js

```diff
@@ -7,7 +7,8 @@ module.exports = function (seconds) {
         throw new TypeError('Invalid value sent to convert-seconds');
     }
     var results = {};
-    results.hours = Math.floor(seconds / 60 / 60);
+    results.days = Math.floor(seconds / 60 / 60 / 24);
+    results.hours = Math.floor(seconds / 60 / 60 -(results.days*24));
```

Comment:
This line is causing the tests to fail:

![image](https://user-images.githubusercontent.com/5614571/37282514-f817e3c6-25f4-11e8-8c5f-631514a312fd.png)

I would leave the hours alone, and let them be calculated as independent values, but it could be interesting to let the return value include a day context, like:

```
hours: 26,
minutes: 12,
seconds: 22,
day: {
	days: 1,
	hours: 2,
	minutes: 12,
 	seconds: 22
}
```

or just give the total number of days:

```
hours: 26,
minutes: 12,
seconds: 22,
totalDaySpan: 1
```

otherwise, I personally like just having the raw number values and keeping it simple, and the users can calculate what they need from the raw totals.